apps/quickbooks_online/models.py

```diff
@@ -201,6 +201,37 @@ def get_department_id_or_none(expense_group: ExpenseGroup, lineitem: Expense = N
     return department_id
 
 
+def get_category_mapping(workspace_general_settings: WorkspaceGeneralSettings, category: str, workspace_id: int):
+    account = None
+    detail_type = ''
+
+    #get the item-mapping if import_items is true
```

Comment:
```suggestion
    # get the item-mapping if import_items is true
```
always add space between `#` and the actual comment